tools/lkl/tests/net.sh

```diff
@@ -43,15 +43,19 @@ fi
 if ! [ -z $DST ]; then
     echo "== RAW socket (LKL net) tests =="
     sudo ip link set dev ${IFNAME} promisc on
-    sudo ./net-test raw ${IFNAME} ${DST} dhcp
+    sudo ./net-test raw ${IFNAME} ${DST} dhcp &
+    sleep 5
+    sudo killall -q net-test && (echo "killed, skipped")
```

Comment:
Maybe we should use an env variable to specificy if DHCP test should run? Then we can set it in those environments where we know it runs.